### DISCUSSION TITLE
Powering ON the PC after it was shut down using WinNUT?

### DISCUSSION THREAD

**Discussion by scalpmonkey** [nutdotnet/WinNUT-Client#95] [Q&A]
Hi,
any ideas what is the best approach how to boot up the PC after it was shut down due power failure using WinNUT Client?

The bios is set to always power but the problem is that for the PC's power was never lost. It was on UPS all the time. 

Tnx.

**Comment by gbakeman** [comment]
This is tricky since your options are somewhat limited. If you had some sort of USB device that you could program to wake the computer, that might be one way. Otherwise you could have another device that sends a "magic packet" (Wake On LAN) to the PC once the power situation has stabilized.

**Reply by scalpmonkey** [reply]
I was hoping there is a way I cannot think of :)
tnx
